Server/Utilities/SpringBoardAlerts.m

```diff
@@ -45,28 +30,97 @@ - (instancetype)init {
+    if (!alertDict[@"title"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No title"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (!alertDict[@"buttons"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No buttons"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (((NSArray *)alertDict[@"buttons"]).count == 0) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"Zero size buttons array"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+    if (!alertDict[@"shouldAccept"]) {
+        @throw [NSException
+                exceptionWithName:@"Bad springboard-alerts JSON"
+                reason: @"No shouldAccept"
+                userInfo:@{@"language":language,
+                           @"alert": alertDict,
+                           @"position":@(position)}];
+    }
+}
+
 - (instancetype)init_private {
     self = [super init];
     if (self) {
-        _alerts = [self concatenateArrays:@[
-                                            [self USEnglishAlerts],
-                                            [self DanishAlerts],
-                                            [self DutchAlerts],
-                                            [self DutchAlerts_BE],
-                                            [self SwedishAlerts],
-                                            [self EUSpanishAlerts],
-                                            [self ES419SpanishAlerts],
-                                            [self GermanAlerts],
-                                            [self FrenchAlerts],
-                                            [self RussianAlerts],
-                                            [self PTBrazilAlerts],
-                                            [self KoreanAlerts],
-                                            [self ItalianAlerts],
-                                            [self HebrewAlerts],
-                                            [self GreekAlerts],
-                                            [self HungarianAlerts],
-                                            [self CzechAlerts]
-                                            ]];
+        NSTimeInterval startTime = [[CBXMachClock sharedClock] absoluteTime];
+        NSBundle * bundle = [NSBundle bundleForClass:[self class]];
+        NSMutableArray<SpringBoardAlert *> * result =
+        [NSMutableArray<SpringBoardAlert *> array];
+        
+        
+        NSArray<NSString*> * languages = @[
```

Comment:
Is it possible to just collect all of programmatically?

If we were not using an asset bundle and just including the files in the bundle, we could use NSBundle to find all the .json files in a directory.